Makefile

```diff
@@ -269,6 +253,25 @@ dist/${PACKAGE}.tar.gz: ${BINARY_PATH} | dist/
 dist/${PACKAGE}.tar.gz.signature.sha256: dist/${PACKAGE}.tar.gz | dist/
 	openssl dgst -sha256 -sign $(PRIVATE_KEY_FILE) -passin pass:"$(PRIVATE_KEY_PASSPHRASE)" $^ | openssl base64 -out $@
 
+################################################################################
+# Target: swagger-docs
+################################################################################
```

Comment:
I thought the intent was to remove this?

Reply:
We still generate the `swagger.json` file as its used to generate our Python SDK. But we no longer generate the `docs/docs.go` file.